<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogExportOptions</class>
 <widget class="QDialog" name="DialogExportOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export options</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="exportSelection">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Entries to export</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="radioButtonSel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonAll">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSelHidden">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected with hidden</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogExportOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogExportOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
